matsim/scenariogen/data/formats/eodmx.py

```diff
@@ -0,0 +1,423 @@
+# -*- coding: utf-8 -*-
+import math
+import os
+import random
+import sys
+
+import pandas as pd
+import numpy as np
+
+from .. import *
+
+# Has households (hogares), persons, trips and viviendas
```

Comment:
What is the difference between hogares and viviendas ?

Reply:
Techinally, both can be translated to "household". A look into the data reveals that almost all hogar <-> vivienda relations are 1:1 (~93%). There are some viviendas, which include multiple hogares. Thus: Generally spoken, hogar and vivienda are the same thing: A household. In cases, where there are multiple hogares inside of one vivienda, the hogares can be viewed as a "sub-household" or something like a core family. I hope this explanation helps, I also struggled a lot with those 2 categories.